.ci/build-kit/compile.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+# Install coverage dependencies
+apt update && apt install -y gcovr lcov
+
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source "$HOME/.cargo/env"
+
+cmake \
+    -B build \
+    -S "$EXT_MOUNT/source" \
+    -G Ninja \
+    -DEVC_ENABLE_CCACHE=1 \
+    -DISO15118_2_GENERATE_AND_INSTALL_CERTIFICATES=OFF \
+    -DBUILD_TESTING=ON \
+    -DEVEREST_ENABLE_RS_SUPPORT=ON
+
+ninja -j$(nproc) -C build && ninja test -C build
```

Comment:
Just concerning the naming here: it is called compile, but actually it also runs the tests.  I would either split it into two or call it *compile and test* or something similar